QuickWKT.py

```diff
@@ -239,7 +239,7 @@ def save_wkt(self, wkt, layerTitle=None):
                         continue
 
                     geom = QgsGeometry.fromWkt(wktLine)
-                    if not geom:
+                    if not geom and not geom.isEmpty():
```

Comment:
The second condition is asking for some attribute of a `not geom` (first part). Were you maybe thinking of this?
```suggestion
                    if not geom or geom.isEmpty():
```

Reply:
No, `geom.isEmpty()` is a valid geometry. It's the scope of this PR!